### DISCUSSION TITLE
Is there documentation such as KDoc or JavaDoc?

### DISCUSSION THREAD

**Discussion by s12works** [littlektframework/littlekt#207] [Q&A] [answered]
I was just wondering if there is any documentation other than [littlekt.com/docs](https://littlekt.com/docs/) (such as Kdoc or JavaDoc).

My computer is not powerful enough to run IntelliJ IDEA with a Kotlin language server, so I cannot use components from the library easily. I have to know their "package location".

I apologize if this question is unworthy of a discussion on GitHub. I am new to Gradle and tools such as KDoc and JavaDoc. There could be a conventional method of acquiring this documentation that I am unaware of.

Thank you for your help; it is greatly appreciated.

Also I am very excited about this framework and its future; it looks awesome!

**Comment by LeHaine** [comment, marked as answer]
Hey! That is a good question. Unfortunately, there isn't anything available at the moment. But let me see what I can do and see if I can get something generated and up and running.

**Reply by LeHaine** [reply]
@s12works Try this: https://littlekt.com/dokka/

**Reply by s12works** [reply]
Wow, thanks for taking the time to do this!

This helps a lot. I really appreciate it!
